modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAuthCodeGrantOpenIdTestCase.java

```diff
@@ -369,7 +369,10 @@ private void changeISConfiguration() throws Exception {
     private void resetISConfiguration() throws Exception {
 
         log.info("Replacing identity.xml with default configurations");
-        serverConfigurationManager.restoreToLastConfiguration();
+        File defaultIdentityXml = new File(getISResourceLocation() + File.separator + "default-identity.xml");
```

Comment:
Can't we move this and accompanying lines to a util class? It seems like this is repeated in several test cases